<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">QText</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>QText.ico</normaloff>QText.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidgetEx" name="tabWidget" native="true">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionRename"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionGoto"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/new.png</normaloff>:/icons/16x16/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New file (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/save.png</normaloff>:/icons/16x16/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save file (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReopen">
   <property name="text">
    <string>Re&amp;open</string>
   </property>
   <property name="toolTip">
    <string>Reload file (Ctrl+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/rename.png</normaloff>:/icons/16x16/rename.png</iconset>
   </property>
   <property name="text">
    <string>R&amp;ename</string>
   </property>
   <property name="toolTip">
    <string>Rename file (F2)</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionShowContainingDirectory">
   <property name="text">
    <string>Show Containing &amp;Directory</string>
   </property>
  </action>
  <action name="actionShowContainingDirectoryOnly">
   <property name="text">
    <string>Show Containing &amp;Directory</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/cut.png</normaloff>:/icons/16x16/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="toolTip">
    <string>Cut text (Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/copy.png</normaloff>:/icons/16x16/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy text (Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/paste.png</normaloff>:/icons/16x16/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste text (Ctrl+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/undo.png</normaloff>:/icons/16x16/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo action (Ctrl+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/redo.png</normaloff>:/icons/16x16/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo action (Ctrl+Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/delete.png</normaloff>:/icons/16x16/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete file</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="icon">
    <iconset resource="../qtext.qrc">
     <normaloff>:/icons/16x16/goto.png</normaloff>:/icons/16x16/goto.png</iconset>
   </property>
   <property name="text">
    <string>Go &amp;To</string>
   </property>
   <property name="toolTip">
    <string>Find folder or file (Ctrl+G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTabWidgetEx</class>
   <extends>QWidget</extends>
   <header>qtabwidgetex.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qtext.qrc"/>
 </resources>
 <connections/>
</ui>
